shared/src/vm/host_env.rs

```diff
@@ -1997,6 +2053,18 @@ where
     Ok(())
 }
 
+/// Set the sentinel for an invalid tx section commitment
+pub fn tx_set_commitment_sentinel<MEM, DB, H, CA>(env: &TxVmEnv<MEM, DB, H, CA>)
```

Comment:
let's have this return `TxResult<()>` (even though it's infallible, the consistency it will avoid having a special macro case for it in tests)

Reply:
actually, I take it back - it would make the usage awkward when we `.map_err` which is more important